docs/how_to/single_node.sh

```diff
@@ -0,0 +1,100 @@
+# Code generated by doctest. DO NOT EDIT.
```

Comment:
fine for this PR, we'll move this script to another directory in the followup PR when we integrate this into the CI process